fenix/app/src/main/java/org/mozilla/fenix/debugsettings/ui/DebugOverlay.kt

```diff
@@ -36,7 +36,7 @@ fun DebugOverlay() {
         modifier = Modifier.fillMaxSize(),
     ) {
         FloatingActionButton(
-            icon = painterResource(R.drawable.ic_new),
+            icon = painterResource(R.drawable.ic_debug_transparent_fire_24),
```

Comment:
I'm not seeing that we include the size of the icon in our other drawables. Do we need it here and can we remove the `_24` if not?

Reply:
so the icons in fenix don't typically have the `_24`, but all of the icons in AC have it. I believe the preference going forward it to include it

Reply:
Gotcha, thanks!